README.md

```diff
@@ -1,56 +1,80 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Movie Rating Website
 
-> Description the project.
+> This is a website that consumes TV maze API and allows users to like and post reviews of movies they have watched. It is a capstone project for JavaScript and Networking Module two of the microverse curricullum.
 
+## [Watch a video description of the features](https://drive.google.com/file/d/1ysVUkkB0vx2lSc_-8Gol2e5RUtHi2y5R/view?usp=sharing)
```

Comment:
- [ ] Nice job creating a demo video about your project features, however, please also add the following to your video:

- All the members in the video should talk about the project i.e your partner.
- The video should contain a highlight about a piece of code that you find interesting or are proud of making.